library/xmltraduit.sax.po

```diff
@@ -0,0 +1,288 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2001-2016, Python Software Foundation
+# This file is distributed under the same license as the Python package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Python 3.6\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-08-10 00:49+0200\n"
+"PO-Revision-Date: 2017-11-20 13:52+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: ../Doc/library/xml.sax.rst:2
+msgid ":mod:`xml.sax` --- Support for SAX2 parsers"
+msgstr ":mod:`xml.sax` --- Support SAX2 pour analyseurs"
+
+#: ../Doc/library/xml.sax.rst:11
+msgid "**Source code:** :source:`Lib/xml/sax/__init__.py`"
+msgstr "**Code source :** :source:`Lib/xml/sax/__init__.py`"
+
+#: ../Doc/library/xml.sax.rst:15
+msgid ""
+"The :mod:`xml.sax` package provides a number of modules which implement the "
+"Simple API for XML (SAX) interface for Python.  The package itself provides "
+"the SAX exceptions and the convenience functions which will be most used by "
+"users of the SAX API."
+msgstr ""
+"The :mod: le paquet 'xml.sax' fournit des modules qui implementent l'interface "
```

Comment:
*The* ?
*:mod:* et *\`xml.sax\`* doivent rester collés.

Reply:
Je te laisse relire tes traductions il semble que tu aies plusieurs fois le problème de tag restructuredtext. la syntaxe des tags est :

    :tagname:`content`

et ne peut pas changer durant la traduction, c'est un tag restructuredtext, le parseur doit pouvoir le retrouver et le comprendre.

Reply:
Bonjour, 
J'ai revu les erreurs présentes dans la première version de ma traduction du fichier et j'ai essayé de les corriger.
Cordialement .